libs/dexie-svelte-query/src/lib/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./stateQuery.svelte.js";
```

Comment:
Not sure, but this should have typescript file suffix or?

```suggestion
export * from "./stateQuery.svelte.ts";
```

Reply:
Typescript empraces the .js suffix despite the source being ts so .js should be the correct suffix